mantidproject/mantid
# Improve masking behaviour of `Display Mask` in ISIS SANS GUI

Fixes #20237

This PR ensures that masked workspaces are shown correctly when using the `Display Mask` functionality of the mask tab in the ISIS SANS GUI. Note that this is in the OLD gui


**To test:**

Please find test data here: smb://olympic/babylon5/Scratch/Anton/GUI_TEST

1. Open the ISIS SANS Gui
1. Select `SANS2D` as the instrument
2. For the user file load: USER_SANS2D_154E_2p4_4m_M3_Xpress_8mm_SampleChanger_FRONT.txt
3. For sample scatter select `SANS2D00034484`
4. Press `Load`
5. Go to the mask tab and press `Display Mask`
    * Confirm that the InstrumentView opens and that the masked areas are displayed in a grey color

**Release Notes** 
See [here](https://github.com/mantidproject/mantid/pull/20308/commits/5640e7017990fdc4b611b9486ef65bfb4002c430)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
